sda-commons-server-opentelemetry/src/main/java/org/sdase/commons/server/opentelemetry/autoconfig/SdaConfigPropertyProvider.java

```diff
@@ -0,0 +1,39 @@
+package org.sdase.commons.server.opentelemetry.autoconfig;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This config source provider is used to define custom defaults that are relevant for sda services.
+ * A complete list of system properties that can be overridden can be found in <a
+ * href="https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md">Autoconfig
+ * module.</a>
+ */
+public final class SdaConfigPropertyProvider {
+  private SdaConfigPropertyProvider() {
+    // avoid creating instances
+  }
+
+  private static final String JAEGER_SERVICE_NAME_ENV_VAR = "JAEGER_SERVICE_NAME";
+
+  public static Map<String, String> getProperties() {
+    Map<String, String> properties = new HashMap<>();
+
+    // default otel collector gateway endpoint 4317 used for grpc
+    properties.put("otel.exporter.otlp.endpoint", "http://otel-collector-gateway.jaeger:4317");
```

Comment:
How can these properties be overridden? Looks very custom.

Reply:
In the tryout deployment we where able to configure it with `OTEL_EXPORTER_JAEGER_ENDPOINT=http://jaeger-collector.jaeger:14250` and the default defined here did not work.

Should we use the default of our dev cluster here or should we omit the setting in the code?

- ```suggestion
      properties.put("otel.exporter.otlp.endpoint", "http://jaeger-collector.jaeger:14250");
  ```
- ```suggestion
  ```

We also should remember, to update the deployment docs in our service templates when we upgrade them, to document the options of open telemetry properly.